### PR TITLE
[Mailer] Explaining In-Reply-To and References header

### PR DESCRIPTION
**Pull request by ThomasLandauer**
* ... as announced in https://github.com/symfony/symfony-docs/issues/13869 ;-)
* Other change: "autoresponder" seems to be the more common term than "autoreplier": https://en.wikipedia.org/wiki/Autoresponder

Closes https://github.com/symfony/symfony-docs/issues/13869

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -322,10 +322,12 @@ header, etc.) but most of the times you'll set text headers::
 
     $email = (new Email())
         ->getHeaders()
-            // this header tells auto-repliers ("email holiday mode") to not
+            // this non-standard header tells compliant autoresponders ("email holiday mode") to not
             // reply to this message because it's an automated email
             ->addTextHeader('X-Auto-Response-Suppress', 'OOF, DR, RN, NRN, AutoReply');
```
Comment (xabbuh) [inline mailer.rst]: the semicolon here needs to be removed

File: mailer.rst [hunk 2]
```diff
@@ -322,10 +322,12 @@ header, etc.) but most of the times you'll set text headers::
...
-
+            // to add multiple  values for an `In-Reply-To` or `References` header
+            // you can pass an array
```
Comment (xabbuh) [inline mailer.rst]: I would generelise this a bit:

> use an array if you want to use a header with multiple values (for example in the "References" or "In-Reply-To" header)

File: mailer.rst (hunk 1 above)
Comment (ThomasLandauer) [reply mailer.rst]: Done :-)

File: mailer.rst (hunk 2 above)
Comment (ThomasLandauer) [reply mailer.rst]: Done :-)

**Comment by javiereguiluz** [comment]
Thank you Thomas.

**Pull request merged**